docs/resources/elb_l7rule.md

```diff
@@ -19,21 +21,190 @@ resource "huaweicloud_elb_l7rule" "l7rule_1" {
 }
 ```
 
+### Create by conditions and type is HOST_NAME
+
+```hcl
+variable l7policy_id {}
+
+resource "huaweicloud_elb_l7rule" "l7rule_1" {
+  l7policy_id  = var.l7policy_id
+  type         = "HOST_NAME"
+  compare_type = "EQUAL_TO"
+
+  conditions {
+    value = "test.com"
+  }
+}
+```
+
+### Create by conditions and type is PATH
+
+```hcl
+variable l7policy_id {}
+
+resource "huaweicloud_elb_l7rule" "l7rule_1" {
+  l7policy_id  = var.l7policy_id
+  type         = "PATH"
+  compare_type = "STARTS_WITH"
+
+  conditions {
+    value = "/test"
+  }
+}
+```
+
+### Create by conditions and type is METHOD
+
+```hcl
+variable l7policy_id {}
+
+resource "huaweicloud_elb_l7rule" "l7rule_1" {
+  l7policy_id  = var.l7policy_id
+  type         = "METHOD"
+  compare_type = "EQUAL_TO"
+
+  conditions {
+    value = "POST"
+  }
+}
+```
+
+### Create by conditions and type is HEADER
+
+```hcl
+variable l7policy_id {}
+
+resource "huaweicloud_elb_l7rule" "l7rule_1" {
+  l7policy_id  = var.l7policy_id
+  type         = "HEADER"
+  compare_type = "EQUAL_TO"
+
+  conditions {
+    key   = "testKey"
+    value = "testValue"
+  }
+}
+```
+
+### Create by conditions and type is QUERY_STRING
+
+```hcl
+variable l7policy_id {}
+
+resource "huaweicloud_elb_l7rule" "l7rule_1" {
+  l7policy_id  = var.l7policy_id
+  type         = "QUERY_STRING"
+  compare_type = "EQUAL_TO"
+
+  conditions {
+    value = "testKey"
+    value = "testValue"
+  }
+}
+```
+
+### Create by conditions and type is SOURCE_IP
```

Comment:
Just keep some typical examples will be fine.

Reply:
done